bin/infrasim-init

```diff
@@ -54,20 +55,21 @@ def create_infrasim_directories():
         os.system("rm -rf /usr/local/libexec")
         os.mkdir("/usr/local/libexec")
 
+
```

Comment:
One thought to function of def create_infrasim_directories(): - Sorry it is not completely relevant.
Should we do things like deleting all those folders if they already exists?
Especially /etc/infrasim, it might contains one yml file which has many changes user want to keep?